cmd/root.go

```diff
@@ -14,6 +16,22 @@ var RootCmd = &cobra.Command{
 	},
 }
 
+var requiredEnvVariables = []string{"SLACK_URL","AHA_USERNAME","AHA_PASSWORD"}
+
 func init() {
-	RootCmd.AddCommand(newCmdAlert())
+    checkEnvVariables(requiredEnvVariables)
+}
+
+func checkEnvVariables(keys []string) error {
```

Comment:
Here the same. Please do not touch cmd, but put the whole logic into pkg/ folder first.